src/grasshopper/lib/grasshopper.py

```diff
@@ -177,3 +179,27 @@ def load_shape(shape_name: str, **kwargs) -> LoadTestShape:
                 f"Shape {shape_name} does not exist in "
                 f"grasshopper.lib.util.shapes! Please check the spelling."
             )
+
+    @staticmethod
+    def set_ulimit():
+        """Increase the maximum number of open files allowed."""
+        # Adapted from locust source code, main function in locust.main.
+        try:
+            minimum_open_file_limit = 10000
```

Comment:
Maybe we should make this value be a command line arg?

Reply:
hmmmm good point but also I think since locust itself doesn't do that, it's probably not something that should be exposed to the user